docs/get-started/installation/prepare/rpi-setup.md

```diff
@@ -35,91 +35,77 @@ If the value of "Architecture: _'xxxxxx'_" ends in "64", you can skip ahead to [
 
 ## Hardware requirements
 
-- A [Raspberry Pi single board computer](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/)
+- A [Raspberry Pi single-board computer](https://www.raspberrypi.com/products/raspberry-pi-4-model-b/)
 - A microSD card
 - An internet-connected computer
 - A way to connect the microSD card to the computer (microSD slot or microSD reader)
 
 ## Install Raspberry Pi OS
 
-To install Raspberry Pi OS (formerly called Raspbian) on a microSD card from which the Pi boots, connect the microSD card to your computer.
+The Raspberry Pi boots from a microSD card.
+You need to install Raspberry Pi OS (formerly called Raspbian) on the microSD card you will use with your Pi:
 
-1.  Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
+1. Connect the microSD card to your computer.
 
-    {{< imgproc alt="Raspberry Pi Imager launcher window showing a 'Choose OS' and 'Choose Storage' buttons." src="/get-started/installation/rpi-setup/imager-launch-screen.png" resize="800x" declaredimensions=true >}}
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
 
-1.  Select `CHOOSE DEVICE`. Choose your model of Pi.
+   {{< imgproc alt="Raspberry Pi Imager launcher window showing a 'Choose OS' and 'Choose Storage' buttons." src="/get-started/installation/rpi-setup/imager-launch-screen.png" resize="800x" declaredimensions=true >}}
 
-    {{< imgproc alt="Raspberry Pi Imager window showing available pi models." src="/get-started/installation/rpi-setup/select-pi-models.png" resize="800x" declaredimensions=true >}}
+1. Select **CHOOSE DEVICE**.
```

Comment:
I feel like these aren't options similar to a drop down, these are each menus so click feels right
```suggestion
1. Click **CHOOSE DEVICE**.
```